_gsdocs-proposals/2020/proposal_AllpixSquaredRestructureDocs.md

```diff
@@ -0,0 +1,57 @@
+* **Infrastructure:** We would like to streamline the way we present the documentation and especially improve its online representation. Optimally we would end up with a streamlined, fully-integrated online presence which coherently presents news, manual and code reference. One possibility for this would e.g. be moving the documentation to [Docsy.dev](https://www.docsy.dev/). The challenging part about this is that we would like to keep as much as possible of our current automatic deployment and workflow.
+* **Bonus task:** For those of you who would like to dive a bit deeper into the scientific depth of this project - we currently do have a few examples for users to understand how to start using the framework and how to use certain features, but they are not very nicely presented and not very extensively documented. They could use a good polishing. Ultimately, these and further examples can be streamlined to follow a tutorial-like workflow.
+
+## Project duration
+
+We are open for both standard-length projects (3 months) and long-running (6 months) projects, depending on what you think is required to achieve the results we are aiming for.
+Given the good overall level of documentation of the project, we think a standard project might suffice.
+
+## Related material
+
+  * Project website: [http://cern.ch/allpix-squared](http://cern.ch/allpix-squared)
+  * Repository of website (hugo): [APSQ Website @ CERn GitLab](https://gitlab.cern.ch/allpix-squared/allpix-squared-website)
+  * Project repository: [Allpix Squared @ CERN GitLab](https://gitlab.cern.ch/allpix-squared/allpix-squared)
+  * Documentation within the repository: [doc/usermanual](https://gitlab.cern.ch/allpix-squared/allpix-squared/-/tree/master/doc%2Fusermanual)
+  * Online documentation: [Online User Manual](https://project-allpix-squared.web.cern.ch/project-allpix-squared/usermanual/allpix-manual.html)
+  * PDF documentation: [PDF User Manual](https://project-allpix-squared.web.cern.ch/project-allpix-squared/usermanual/allpix-manual.pdf)
+  * Doxygen code reference: [Code Reference](https://project-allpix-squared.web.cern.ch/project-allpix-squared/reference)
+
+## Expected results
+
+* A restructured user manual which clearly separates content intended for users and developers
+* A reviewed manual with previously missing information on the project and the framework
+* A streamlined online presence which better integrates project website, documentation and code reference
+
+## Experience required
+Our current documentation workflow consists of a few tools and in order to rip them apart it would be good to have a solid understanding of:
+LaTeX, Doxygen, pandoc, hugo, git, GitLab and its Continuous Integration - as well as of all the new and cool tools that you will introduce to make our documentation sparkle.
+
+## Mentors
+  * [Paul Schütze](mailto:paul.schuetze@desy.de)
```

Comment:
Please, update `gsdocs/2020/mentors.md` with the mentors

Reply:
Done.